From 0b760c0cc5049d2f07391bf099aad1c48f29046c Mon Sep 17 00:00:00 2001
From: Jan Pavlinec <jan.pavlinec@nic.cz>
Date: Fri, 8 Jun 2018 16:04:01 +0200
Subject: [PATCH] memcached: update to version 1.5.8 (security fix)

CVE-2018-1000127 - version prior to 1.4.37 contains an Integer Overflow vulnerability https://security-tracker.debian.org/tracker/CVE-2018-1000127
CVE-2017-9951 -version prior to 1.4.39 allows remote attackers to cause a denial of service (segmentation fault) via a request to add/set a key https://security-tracker.debian.org/tracker/CVE-2017-9951

Issue https://gitlab.labs.nic.cz/turris/openwrt/issues/194
---
 net/memcached/Makefile | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/net/memcached/Makefile b/net/memcached/Makefile
index bd5267a..7e70ce3 100644
--- a/net/memcached/Makefile
+++ b/net/memcached/Makefile
@@ -9,12 +9,12 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=memcached
-PKG_VERSION:=1.4.31
+PKG_VERSION:=1.5.8
 PKG_RELEASE:=1
 
 PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
 PKG_SOURCE_URL:=http://memcached.org/files
-PKG_MD5SUM:=c19bb0e77e720f64f33ecb43de28a1b4
+#PKG_HASH:=9ac93113bdb5d037e79c61277386564ac2e5e31d49e594f11e554e4c149b7245
 
 PKG_MAINTAINER:=Thomas Heil <heil@terminal-consulting.de>
 
-- 
2.7.4

